external-crates/move/move-compiler/src/cfgir/optimize/known_jumps.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
A doc comment at the beginning might be helpful (we should really do this more often)